components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -1940,6 +1940,16 @@ public static NativeArray jsFunction_getAPI(Context cx, Scriptable thisObj, Obje
                                     policyObj.put("tierDisplayName", policyObj, policy.getName() != null ? policy.getName() : "");
                                     policyObj.put("tierDescription", policyObj,
                                             policy.getDescription() != null ? policy.getDescription() : "");
+                                    if (policy.getTierAttributes() != null) {
+                                        Map<String, Object> attributes;
+                                        attributes = policy.getTierAttributes();
+                                        StringBuilder attributesListBuilder = new StringBuilder();
+                                        for (Map.Entry<String, Object> attribute : attributes.entrySet()) {
+                                            attributesListBuilder.append(attribute.getKey()).append("::").append(
+                                                    attribute.getValue()).append(",");
+                                        }
+                                        policyObj.put("tierAttributes", policyObj, attributesListBuilder.toString());
```

Comment:
Shouldn't you remove the last comma of `attributesListBuilder.toString()` at this point?